### PR TITLE
Update k8s.jsonnet file to support 2023 changes to images

### PR DESCRIPTION
**Pull request by HubbeKing**
These are the changes I made to make it actually function for the year's run, but I would like it looked over in case I did an Oopsie somewhere in it.

Could be that's why my node's metrics where a little Weird this year, after all.

### CODE REVIEW

File: k8s.jsonnet [hunk 1]
```diff
@@ -109,26 +113,26 @@
     // May be the same as cutter_creds_file.
     sheetsync_creds: import "./google_creds.json",
 
+    // Path to a file containing a twitch OAuth token to use when downloading streams.
+    // This is optional (null to omit) but may be helpful to bypass ads.
+    downloader_creds_file: null,
+
     // The URL to write to the sheet for edit links, with {} being replaced by the id
-    edit_url: "https://wubloader.example.com/thrimbletrimmer/edit.html?id={}",
+    edit_url: "http://thrimbletrimmer.codegunner.com/edit.html?id={}",
```
Comment (ekimekim) [inline k8s.jsonnet:121]: i've been trying to remove this old url from the codebase. the example.com one is fine.
```suggestion
    edit_url: "https://wubloader.example.com/thrimbletrimmer/edit.html?id={}",
```

**Review by ekimekim — APPROVED** [review]
all looks reasonable to me

File: k8s.jsonnet (hunk 1 above)
Comment (HubbeKing) [reply k8s.jsonnet]: I have done this.

**Comment by HubbeKing** [comment]
Using some more better mount options does appear to have sped things up considerably, but I suspect we won't know for sure without a proper test with backfill going and stuff.

**Pull request merged**